### ISSUE TITLE
Win10 blocks app when non-admin

### ISSUE THREAD

**Issue by Scrotos** [TumblThreeApp/TumblThree#128]
I just upgraded Win10 to 20H2 and this app stopped working.  I noticed that all the files in the folder were blocked and I had to right-click to go to properties to unblock them.  Double clicking the app sometimes brought up a windows defender box saying the app was unknown and could cause harm, etc, and I could go into more options and run anyway.  Even then, nothing happened.  Sometimes I would double click the app and not get a dialog box but nothing loaded at all, nothing I could see as foreground or background processes seemed to indicate it was running in Task Manager.  

Some policy in Windows must have changed in 20H2, I guess.  And I don't run as an admin-level user.  I don't know if this is a one-off scenario or if it can be duplicated by other folk.  I tried a fresh download into a new folder and the same issue persisted.

**Comment by Scrotos** [comment]
BTW, it does work when I right-click and run as admin.  So... not sure what's up.

**Comment by thomas694** [comment]
We haven't heard of a problem yet. I cannot reproduce it here, but maybe someone else can.

Does anybody else have this problem?

**Comment by thomas694** [comment]
I works well here in 20H2 with a standard user account and a fresh download of the zip file. There's a one-time warning of Defender, but you can click on "More info" and "Run anyway". I cannot test with an already installed app and upgrading to 20H2.

Maybe you have additional problems because your files still had the "block" attributes from before the upgrade. But normally a one-time removal as admin should be enough to unblock them.
If that doesn't help at all, you could still try remove your whole TumblThree folder and start with a fresh downloaded zip file.

Please let us know, if you find the reasons for your described behavior.

**Comment by thomas694** [comment]
@ all
Does someone else also have problems since the Win 10 upgrade to version 20H2?

**Comment by Scrotos** [comment]
Ok, finally got off my butt and tried on a virgin Win10 20H2 system.  As a user in local admins group it gave a warning about file from the internet, blah blah, click more options, then run anyway.  As expected it didn't prompt to run anyway on subsequent runs.  

I made a new user who is only in local users group.  Downloaded app fresh, got the same warning, and then... the damn thing worked.  I tested with 1.4.0 and have not yet tried that version on the affected system.

The users are local and not hooked into Microsoft for cloud accounts.  The one on the test system might be?  I don't know.  But hell, at this point it's probably something dumb I've done to myself since I can't replicate it with similar user privileges to the original system.
